build/media-suite_compile.sh

```diff
@@ -454,7 +454,7 @@ if [[ $ffmpeg != "no" || $standalone = y ]] && enabled libtesseract; then
 
     _check=(libtesseract.{,l}a tesseract.pc)
     if do_vcs "https://github.com/tesseract-ocr/tesseract.git"; then
-        do_pacman_install docbook-xsl nettle
+        do_pacman_install docbook-xsl nettle libarchive
```

Comment:
I think nettle is a libarchive dep, so you can remove it.

Reply:
Ah, OK, I see, "depends on". Thank you, next time I will check before!